tutorials/Intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,448 @@
+---
+title: An Introduction to Geospatial Analysis Using Google Earth Engine
+description: This workshop provides a fundamental overview of remote sensing and geospatial analysis using the JavaScript API of Earth Engine.
+author: TC25
+tags: introductory, GIS, remote sensing, Google Earth Engine
+date_published: 2019-09-14
+---
```

Comment:
Is there also a single complete source sample you can submit in a separate file? (e.g. sample.js)

Reply:
I added a sample.js file with multiple scripts from the code editor